### DISCUSSION TITLE
Double chest support and 1.19.2 backwards compatibility

### DISCUSSION THREAD

**Discussion by tacogerbil** [StoneLabs/working-graves#10] [Q&A] [answered]
Is the 1.19.4 backwards compatible with 1.19.2? (Still have to use it for Cobblemon :S).
and does this support double chests?

**Comment by StoneLabs** [comment, marked as answer]
Double chests are fine. (Actually any amount of chests or barrels is fine)

I'm not sure if it will work with 1.19.2. I gave it a quick test and it seams to be fine.
Open the jar with 7-zip and edit `fabric.mod.json`. Replace `"minecraft": ">=1.19.4",` in line 31 with `>=1.19.2`.
However, no guarantees that it wont crash your game at some point ^^

**Comment by tacogerbil** [comment]
Is it supposed to light everything on fire???? lol... When I died there was lightning and fire all around my death chest....  I'm afraid to try again since it chose a different chest than the one I had intended (that was also nearby)...  I don't want to burn down my house!! lol

**Reply by StoneLabs** [reply]
The items drop on the ground near the sign If there is no chest under the grave. They may light on fire from the lightning but won't burn away. (Also they won't despawn like regular items).

I'm pretty sure it won't create actual fire blocks (only burning items), so your house is safe :)

**Comment by tacogerbil** [comment]
This is server safe right?  My nephew created one, killed himself to make sure.. it worked.  He went of looking around for ores, died and his stuff went into my sons death chest, not his own ( my son had not died yet in order for the sign to take his name so it was still underlined "hic portus animae")....
I'm so confused....

**Reply by StoneLabs** [reply]
Once you died and the grave is populated it will not be chosen again. They are single use so to speak.
Thats by design, so you can create a graveyard that will slowly fill up.

Hope that makes sense :)

**Reply by tacogerbil** [reply]
Yes, THANK YOU..
So to make sure... It's random?  If my son makes a sign and my nephew doesn't.. and my nephew dies he'll take the chest/sign my son created?

**Comment by tacogerbil** [comment]
Um..... It was burning down my house lol... Everything was made of wood.  A
big section of my home burnt up 😵‍💫 hahahaha... Man, I was running around
pounding out the fire.  I had to replace a big section around the chest
where the blocks were destroyed.  A couple of items that were supposed to
go into the death chest were destroyed....
Is there a config to disable the fire?

**Reply by StoneLabs** [reply]
Haha sry about that. There is no option to disable the lightning as of yet, but I'll add one in the future :)

**Reply by tacogerbil** [reply]
![YayYesGIF](https://github.com/StoneLabs/working-graves/assets/12961907/af637b8a-c04a-4a92-b816-91bd8fa3edf6)

**Comment by tacogerbil** [comment]
That is actually pretty cool.... Thank you!  If I'm running out of space I
should be able to destroy the sign and reuse the chest with a new sign?

**Reply by StoneLabs** [reply]
Yup, replace the sign. If it's underlined it's good to go.

**Reply by tacogerbil** [reply]
THANK YOU!
